jenkins/automation/kubernetes/deploy-cortx-k8-cluster.groovy

```diff
@@ -87,6 +87,19 @@ pipeline {
                 '''
             }
         }
+
+        stage ('IO Sanity Test') {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                sh label: 'Deploy CORTX Components', script: '''
```

Comment:
Please rename label as `IO Sanity` and remove `Deploy CORTX Components`.

Reply:
Done.